cdk.json

```diff
@@ -33,7 +33,7 @@
 
     "atat:CspConfigurationName": "config/csp/integration",
     "atat:GitHubPatName": "auth/github/pat",
-    "atat:VersionControlBranch": "develop",
+    "atat:VersionControlBranch": "feature/remove-cdk-monitoring-stack",
```

Comment:
Is this a mistake?  Should this remain as _develop_?

Reply:
addressed/fixed